js/dataframe/shared/Table.svelte

```diff
@@ -663,6 +669,7 @@
 						<th
 							class:editing={header_edit === i}
 							aria-sort={get_sort_status(value, sort_by, sort_direction)}
+							style="width: {column_widths[i]};"
```

Comment:
```suggestion
style:width={column_widths.length ? column_widths[i] : undefined}
```
So no rule is applied if there's no column_widths[i]